app/models/staged/sponsors.rb

```diff
@@ -19,8 +17,8 @@ def valid?
     def sponsors_valid?
       # Ask them all to validate...
       sponsors.each { |sponsor| sponsor.valid? }
-      # ...return if any have an error
-      sponsors.any? { |sponsor| sponsor.errors.any? }
+      # ...return true if there are no errors at all
+      sponsors.all? { |sponsor| sponsor.errors.none? }
```

Comment:
I thought so, but it would shortcut and stop at the first ``false``, whereas I want to explicitly call ``valid?`` on each sponsor, so that if we get all the errors for displaying in the UI.